DocGen4/Output/Inductive.lean

```diff
@@ -15,25 +15,36 @@ def instancesForToHtml (typeName : Name) : BaseHtmlM Html := do
         <ul class="instances-for-enum"></ul>
     </details>
 
-def ctorToHtml (c : Process.NameInfo) : HtmlM Html := do
+def ctorToHtml (c : Process.NameInfo) (backrefs : Array BackrefItem) :
```

Comment:
Similar pattern here, all of this reference pull through should be a state monad.